src/main/java/com/arcaniax/gobrush/GoBrushPlugin.java

```diff
@@ -105,6 +106,14 @@ public void onEnable() {
                 return "201+";
             }
         }));
+
+        try {
+            Class.forName("org.bukkit.generator.WorldInfo");
+            getLogger().info("WorldInfo class found! Setting to use new world limits.");
+            BlockUtils.setNewerWorldVersion(true);
+        } catch (ClassNotFoundException ignored) {
+            getLogger().info("WorldInfo class not found! Setting to original world limits.");
+        }
```

Comment:
```suggestion
        try {
            Class.forName("org.bukkit.generator.WorldInfo");
            BlockUtils.setNewerWorldVersion(true);
        } catch (ClassNotFoundException ignored) {
        }
```
Don't think we really need the loggers for that.

Reply:
Removed loggers.